File: backend/Modulos/models/estructuraBodega.go
package models

// EstructuraBodega is...
type EstructuraBodega struct {
	Accion            string                            `json:"accion"`
	HDGCodigo         int                               `json:"hdgcodigo"`
	ESACodigo         int                               `json:"esacodigo"`
	CMECodigo         int                               `json:"cmecodigo"`
	CodBodega         int                               `json:"codbodega"`
	DesBodega         string                            `json:"desbodega"`
	FbodEstado        string                            `json:"estado"`
	DespachaReceta    string                            `json:"despachareceta"`
	FbodModificable   string                            `json:"modificable"`
	FbodTipoPorducto  string                            `json:"tipoproducto"`
	FbodTipoBodega    string                            `json:"tipobodega"`
	FbodFraccionable  string                            `json:"fbodfraccionable"`
	GlosaTipoBodega   string                            `json:"glosatipobodega"`
	GlosaTipoProducto string                            `json:"glosatiproducto"`
	FboCodigoBodega   string                            `json:"fbocodigobodega"`
	Servidor          string                            `json:"servidor"`
	Usuario           string                            `json:"usuario"`
	Servicios         []EstructuraServicioUnidadBodegas `json:"serviciosunidadbodega"`
	Productos         []EstructuraProductosBodegas      `json:"productosbodega"`
	Usuarios          []EstructuraUsuariosBodegas       `json:"usuariosbodega"`
	RelacionBodegas   []EstructuraRelacionBodega        `json:"relacionbodegas"`
	ProductosDemanda  []EstructuraProductosDemanda      `json:"productosdemanda"`
}

File: frontend/src/app/models/entity/estructura-bodega.ts
import {ServicioUnidadBodegas} from "../entity/servicio-unidad-bodegas"
import {ProductosBodegas} from "../entity/productos-bodegas"
import {UsuariosBodegas} from "../entity/usuarios-bodegas"
import { EstructuraRelacionBodega } from './estructura-relacion-bodega'


 
export class EstructuraBodega {
    accion       : string;
	hdgcodigo    : number;
	esacodigo    : number;
	cmecodigo    : number;
	codbodega    : number;
	desbodega    : string;
	despachareceta : string;
	estado       : string;
	modificable  : string;
	tipoproducto : string;
	tipobodega   : string;
	glosatipobodega : string;
	glosatiproducto : string;
	servidor        : string;
	fbocodigobodega	: string;
	serviciosunidadbodega : ServicioUnidadBodegas[];
	productosbodega  : ProductosBodegas[];
	usuariosbodega   : UsuariosBodegas[];
	relacionbodegas  : EstructuraRelacionBodega[];
	fbodfraccionable : string;
	bloqueo : string;
}
